### ISSUE TITLE
Resuming from Sleep with a changed display resolution

### ISSUE THREAD

**Issue by sound-fx** [spyder-ide/spyder#13724]
### What steps will reproduce the problem?

Spyder still running on a laptop connected to an external monitor and a network, when I put the laptop to Sleep. I disconnected the external monitor and unplugged the machine from the network. Upon waking the laptop from Sleep, this error message appeared.

### Traceback
```python-traceback
  File "C:\Users\greg\Anaconda3\envs\afcap2\lib\threading.py", line 801, in __bootstrap_inner
    self.run()
  File "C:\Users\greg\Anaconda3\envs\afcap2\lib\site-packages\watchdog\observers\api.py", line 146, in run
    self.queue_events(self.timeout)
  File "C:\Users\greg\Anaconda3\envs\afcap2\lib\site-packages\watchdog\observers\read_directory_changes.py", line 75, in queue_events
    winapi_events = read_events(self._handle, self.watch.is_recursive)
  File "C:\Users\greg\Anaconda3\envs\afcap2\lib\site-packages\watchdog\observers\winapi.py", line 346, in read_events
    buf, nbytes = read_directory_changes(handle, recursive)
  File "C:\Users\greg\Anaconda3\envs\afcap2\lib\site-packages\watchdog\observers\winapi.py", line 306, in read_directory_changes
    raise e
WindowsError: [Error 64] The specified network name is no longer available.

WARNING:spyder.widgets.github.backend:Failed to send bug report on Github. response={'json': {'documentation_url': u'https://docs.github.com/rest', 'message': u'Bad credentials'}, 'code': 401}
```

## Versions

* Spyder version: 4.0.1 
* Python version: 2.7.18
* Qt version: 5.6.2
* PyQt5 version: 5.6
* Operating System: Windows 10

### Dependencies

```
atomicwrites >=1.2.0         :  1.4.0 (OK)
chardet >=2.0.0              :  3.0.4 (OK)
cloudpickle >=0.5.0          :  1.2.2 (OK)
diff_match_patch >=20181111  :  20200713 (OK)
intervaltree                 :  None (OK)
IPython >=4.0;<6.0           :  5.8.0 (OK)
jedi =0.14.1                 :  0.14.1 (OK)
nbconvert >=4.0              :  5.6.1 (OK)
numpydoc >=0.6.0             :  1.1.0 (OK)
pexpect >=4.4.0              :  4.7.0 (OK)
pickleshare >=0.4            :  0.7.5 (OK)
psutil >=0.3                 :  5.6.3 (OK)
pygments >=2.0               :  2.5.2 (OK)
pylint >=0.25                :  1.9.2 (OK)
pyls >=0.31.2;<0.32.0        :  0.31.2 (OK)
zmq >=17                     :  18.1.0 (OK)
qdarkstyle >=2.7             :  2.8.1 (OK)
qtawesome >=0.5.7            :  0.7.2 (OK)
qtconsole >=4.6.0            :  4.7.5 (OK)
qtpy >=1.5.0                 :  1.9.0 (OK)
rtree >=0.8.3                :  0.8.3 (OK)
sphinx >=0.6.6               :  1.8.5 (OK)
spyder_kernels >=1.8.1;<2.0.0:  1.8.1 (OK)
watchdog                     :  None (OK)
cython >=0.21                :  None (NOK)
matplotlib >=2.0.0           :  2.2.3 (OK)
numpy >=1.7                  :  1.16.6 (OK)
pandas >=0.13.1              :  0.24.2 (OK)
scipy >=0.17.0               :  1.2.1 (OK)
sympy >=0.7.3                :  None (NOK)
```

**Comment by ccordoba12** [comment]
Hey @sound-fx, thanks for reporting. This problem should be fixed in our latest version. To update please open the Anaconda prompt and run there

    conda update anaconda
    conda install spyder=4.1.5

Please let us know if it works for you.

**Comment by sound-fx** [comment]
Thank you.
Alas, it will be some time before I can confirm whether that fixes it, as conda can't update Spyder in either of my two virtual environments.
One virt-env uses Python 2, but Spyder 4.1.5 requires 3.x.
The other virt-env installed from anaconda 2020.07, and `conda install spyder=4.1.5` finds many conflicts before giving up.
I'll probably need to switch to miniconda, or possibly roll back to an earlier anaconda, to break this impasse and actually test 4.1.5.

**Comment by sound-fx** [comment]
After updating to Spyder 4.1.5 (by updating to Anaconda 2020.11) a few months back, I have not seen this error again.  As I'm fairly confident that some version after 4.0.1 fixed the error so it no longer occurs with 4.1.5, I'm closing this issue. I'll create a new one if it should ever surface again with 4.1.5 or later.
